src/registries/npm-registry.js

```diff
@@ -110,7 +110,7 @@ export default class NpmRegistry extends Registry {
       [false, path.join(getGlobalPrefix(), filename)],
     ];
 
-    const foldersFromRootToCwd = this.cwd.split(path.sep);
+    const foldersFromRootToCwd = this.cwd.split(/[\\/]/g);
```

Comment:
Normalize would work when using posix path on windows, but not the opposite (on posix, backslashes are valid filename characters, so path.posix doesn't convert them) :(

I'm gonna use `slash`, but I don't understand why it does an ASCII check ... filenames can definitely contain special characters, especially when dealing with user folders (oh how many times I've been burned when Windows put my home folder under `C:\Users\Maël` :().